spec/ParseLiveQueryServer.spec.js

```diff
@@ -753,7 +763,7 @@ describe('ParseLiveQueryServer', function () {
     setTimeout(function () {
       expect(client.pushDelete).toHaveBeenCalled();
       done();
-    }, jasmine.ASYNC_TEST_WAIT_TIME);
```

Comment:
Do you think you could rewrite these timed blocks to use async/await instead of being nested?

I know that goes a bit beyond the scope of this PR, but it would certainly make the tests easier to read. So instead of these nests we would just do:

```
await new Promise(resolve => setTimeout(resolve, ASYNC_TEST_WAIT_TIME));
```